tst/EnergyPlus/unit/PlantLoopHeatPumpEIR.unit.cc

```diff
@@ -500,6 +683,7 @@ TEST_F(EnergyPlusFixture, Initialization)
     EXPECT_NEAR(expectedSourceSideMassFlow, state->dataLoopNodes->Node(thisCoolingPLHP->sourceSideNodes.inlet).MassFlowRateMaxAvail, flowTol);
 }
 
+
```

Comment:
Not sure why this extra new line popped in.  Take it out...or leave it...I don't care :)